source/change-streams/tests/unified/change-streams.yml

```
@@ -177,3 +177,163 @@ tests:
                 comment: "comment"
               commandName: getMore
               databaseName: *database0
+
+  - description: "Test unknown operationType MUST NOT err"
+    runOnRequirements:
+      - minServerVersion: "3.6"
+    operations:
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          # using $project to simulate future changes to ChangeStreamDocument structure
+          pipeline: [ { $project: { operationType: "addedInFutureMongoDBVersion", ns: 1 } } ]
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: insertOne
+        object: *collection0
+        arguments:
+          document: {
+            "_id": 1,
+            "a": 1
+          }
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult: {
+          "operationType": "addedInFutureMongoDBVersion",
+          "ns": {
+            "db": "database0",
+            "coll": "collection0"
+          }
+        }
```

We typically do not hard-code database and collection names. Since these are already defined as `*database0` and `*collection0` in the entity definitions, we can reuse those here.

```suggestion
        expectResult:
          operationType: "addedInFutureMongoDBVersion"
          ns: { db: *database0, coll: *collection0 }
```

 Other tests would typically only define `&database0` for the entity identifier and then define another `&databaseName0` anchor for its name, but that wasn't done in this test file for reasons unknown to me.